tests/clone_test.go

```diff
@@ -607,7 +607,7 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 					expectEqualTemplateAnnotations(targetVM, sourceVM)
 				})
 
-				Context("[QUARANTINE] with instancetype and preferences", decorators.Quarantine, func() {
+				Context("with instancetype and preferences", decorators.Quarantine, func() {
```

Comment:
Apparently need to also drop the quarantine decorator, made the same mistake over at https://github.com/kubevirt/kubevirt/pull/10773#discussion_r1402772832

Reply:
Thanks!